person.rb

```diff
@@ -13,6 +13,10 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
```

Comment:
- [ ] After adding this kindly consider making sure the Person class inherits from the Nameable class to meet the project requirements 
hint 👉  update the first line of this file